app/controllers/declaration/AdditionalInformationController.scala

```diff
@@ -128,11 +127,8 @@ class AdditionalInformationController @Inject()(
   private def updateCacheModelAndRedirect(itemId: String, updatedCache: Seq[AdditionalInformation], redirectCall: Call)(
     implicit request: JourneyRequest[_]
   ): Future[Result] =
-    for {
-      _ <- updateCache(itemId, journeySessionId, AdditionalInformationData(updatedCache))
-      _ <- legacyCustomsCacheService
-        .cache[AdditionalInformationData](goodsItemCacheId, formId, AdditionalInformationData(updatedCache))
-    } yield Redirect(redirectCall)
+    updateCache(itemId, journeySessionId, AdditionalInformationData(updatedCache))
```

Comment:
Same here and probably in every controller

Reply:
ok